iodide-project/iodide
# WIP: myriad output handler changes

This is not complete yet - will tag a reviewer when it is. The goal is to clean up the output handlers a bit:

- [x] fix the warning made by the recent changes to the declared variables pane (#565)
- [x] add typed arrays to the output handlers
- [x] finish the test suite associated with reps / handlers
- [x] DOM elements now output the dom element
- [x] ~DOM elements will render as a DOM inspector, akin to a browser DOM inspector~ __we decided that this is less useful right now that the raw DOM element, so we are going to try that out and see what breaks__

To test a DOM element, try this in a cell:

```javascript
var test = document.createElement('h1')
test.innerHTML = 'hiya!'
test
```

To test the new typed array interface, copy this into a cell and run:

```javascript
var typedArray = new Uint32Array([10000, 2, 3, 4])
typedArray
```

And you'll see it returns a normal `arrayHandler` rendered component, appropriately labeled for now.
